app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestPresenter.java

```diff
@@ -84,7 +85,13 @@ public void addCategory(Category category) {
 
     @Override
     public void onNewCategoryAdded(Intent data) {
-        //todo - Add category
+        String categoryName = data.getStringExtra("NAME");
+        String categoryDescription = data.getStringExtra("DESCRIPTION");
```

Comment:
You have these as constants in `AddCategoryActivity`, maybe you can reuse them in this file. @jmccormack200 is it ok to make them constants in the presenter and have the activity use them?